skate-plugin/src/main/kotlin/com/slack/sgp/intellij/tracing/SkateTraceReporter.kt

```diff
@@ -48,13 +47,12 @@ class SkateTraceReporter(private val project: Project, private val offline: Bool
     val loggingClient = lazy {
       okHttpClient.value.newBuilder().addInterceptor(loggingInterceptor).build()
     }
-    if (offline) {
-      NoOpTraceReporter
-    } else {
-      val endPoint =
-        project.tracingEndpoint()
-          ?: throw RuntimeException("Required tracing endpoint to upload analytics")
+    val endPoint = project.tracingEndpoint()
+    if (endPoint != null) {
       SimpleTraceReporter(endPoint, loggingClient)
+    } else {
+      LOG.info("No endpoint set up for tracing")
+      NoOpTraceReporter
```

Comment:
Stop throwing exception here cause unit test code started making network request to send up the trace. The endpoint should be set up when installing the plugin but empty for unit test because we don't have skate.xml file set up in unit test environment.